server/routes/referrals/referralFormPresenter.ts

```diff
@@ -148,48 +153,127 @@ export default class ReferralFormPresenter {
         ],
       }
     }
-    const checkYourAnswers: ReferralFormSingleListSectionPresenter = {
-      type: 'single',
-      title: 'Check your answers',
-      number: '3',
-      status: this.calculateStatus(this.sectionValues.checkYourAnswers, referralDetails.status),
-      tasks: [
-        {
-          title: 'Check your answers',
-          url: this.calculateTaskUrl('check-answers', this.taskValues.furtherInformation),
-        },
-      ],
-    }
-    return [reviewServiceUserInformation, referralDetails, checkYourAnswers]
-  }
 
-  private calculateTaskUrl(url: string | null, displayCriteria: DraftReferralValues): string | null {
-    const everyFieldHasValues = displayCriteria.every(field => {
-      if (field === null) {
-        return false
+    private buildCohortReferralDetailsSection(
+      selectServiceCategoriesSection: ReferralFormSingleListSectionPresenter
+    ): ReferralFormMultiListSectionPresenter {
+      return {
+        type: 'multi',
+        title: 'Add intervention referral details',
+        number: '3',
+        status: this.calculateStatus(
+          this.sectionValues.serviceCategoryReferralDetails,
+          selectServiceCategoriesSection.status!
+        ),
+        taskListSections: [
+          {
+            tasks: [
+              {
+                title: `Confirm the relevant sentence for the intervention referral`,
```

Comment:
i know this is just moved code, but i wondered whether 'intervention referral' here should be `${this.intervention.contractType.name}` referral, to be consistent with this pattern elsewhere.

Reply:
Sure I will change it also in the section title as well ->
Add ${this.intervention.contractType.name} referral details

Reply:
done